### PR TITLE
Solution for issue 140: Lock down locale.

### PR DESCRIPTION
**Pull request by ovelindstrom**
Lock down the locale to that test will succeed regardless of JVM default Locale.

https://github.com/prometheus/client_java/issues/140

### CODE REVIEW

**Comment by brian-brazil** [comment]
Thanks for the PR, there are several other calls to this function in the codebase. Are any of the rest of them affected?

**Comment by ovelindstrom** [comment]
Could be. Let me take a look.

**Comment by ovelindstrom** [comment]
As far as I can see, no other calls to the getSampleValue in this codebase is affected. Mostly, you use string representation of the quantiles like "0.99" where you query the registy. In this case, the doubles are used to make the arithmetic easier and make the test case shorter.

Lurking in the code of the History and Sample classes, they never actually mention a label format when it comes to different locales. Maybe it would be a good idea to define the used quantile lables.

**Comment by brian-brazil** [comment]
Looking at the other code, this should probably just be `b.toString`

**Comment by ovelindstrom** [comment]
You are right @brian-brazil. simplifying it to use a Double instead of the primitive and using String.format will make it easier. Committing the change.

**Comment by brian-brazil** [comment]
Could you remove the extra import and squash your commit?

**Comment by ovelindstrom** [comment]
Done.

File: simpleclient_dropwizard/src/test/java/io/prometheus/client/dropwizard/DropwizardExportsTest.java [hunk 1]
```diff
@@ -9,9 +9,7 @@
 import java.io.IOException;
 import java.util.Arrays;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (brian-brazil) [inline simpleclient_dropwizard/src/test/java/io/prometheus/client/dropwizard/DropwizardExportsTest.java]: Please leave imports explicit

**Comment by brian-brazil** [comment]
Thanks

**Pull request merged**